test/crowdsale/TimedCrowdsale.test.js

```diff
@@ -20,7 +20,7 @@ contract('TimedCrowdsale', function ([_, investor, wallet, purchaser]) {
   const tokenSupply = new BigNumber('1e22');
 
   before(async function () {
-    // Advance to the next block to correctly read time in the solidity "now" function interpreted by testrpc
+    // Advance to the next block to correctly read time in the solidity "now" function interpreted by ganache
```

Comment:
So we've obviously copy-pasted these test files over and over again 😅 can we just remove these duplicate comments in all of these crowdsale tests? I could also see an argument for the extra verbosity being worth it.